undertow/src/main/java/org/wildfly/extension/undertow/UndertowService.java

```diff
@@ -129,6 +129,10 @@ public static ServiceName accessLogServiceName(final String server, final String
         return virtualHostName(server, virtualHost).append(Constants.ACCESS_LOG);
     }
 
+    public static ServiceName activeRequestTrackingServiceName(final String server, final String virtualHost) {
+        return virtualHostName(server, virtualHost).append(Constants.ACTIVE_REQUEST_TRACKER);
+    }
+
```

Comment:
Since you have a capability defined for this service, we should prefer that for generating the service name.